docs/content/docs/guide/authoringprs.md

```diff
@@ -153,6 +153,7 @@ The fields available are :
 - `headers`: The full set of headers as passed by the Git provider. (example: `headers['x-github-event']` will get the event type on GitHub)
 - `.pathChanged`: a suffix function to a string which can be a glob of a path to
   check if changed (only `GitHub` and `Gitlab` provider is supported)
+- `files`: The list of files that changed in the event (all, added, deleted, modified and renamed). Example `files.all` or `files.deleted`. On pull request every file belonging to the pull request will be listed.
```

Comment:
Can we add a note like this is not supported for Bitbucket yet

Reply:
Good catch, I will update it